pyproject.toml

```diff
@@ -15,6 +15,7 @@ python = ">=3.8,<4"
 sqlalchemy = "^1.3.12"
 pyjwt = "^2.8.0"
 connect-eaas-core = "^30.3"
+fastapi-pagination = "<=0.12.17"
```

Comment:
This is pretty scary, maybe you were considering >= ?
Anyway, for now it's fine